src/main/flight/pid.c

```diff
@@ -597,6 +604,12 @@ void pidInitConfig(const pidProfile_t *pidProfile)
     useIntegratedYaw = pidProfile->use_integrated_yaw;
     integratedYawRelax = pidProfile->integrated_yaw_relax;
 #endif
+
+#ifdef USE_THRUST_LINEARIZATION
+    thrustLinearization = pidProfile->thrustLinearization / 100.0f;
+    thrustLinearizationReciprocal = 1.0f / thrustLinearization;
+    thrustLinearizationB = (1.0f - thrustLinearization) / (2.0f * thrustLinearization);
```

Comment:
This line and the previous need to be bounded by a zero check otherwise they will cause division by zero when the feature is turned off (`thrust_linear = 0`).

Reply:
Excellent catch! Thanks. Done.